chain/client/src/stateless_validation/partial_witness/partial_witness_tracker.rs

```diff
@@ -42,7 +42,7 @@ enum AccessedContractsState {
     Unknown,
     /// Received `ChunkContractAccesses` and sent `ContractCodeRequest`,
     /// waiting for response from the chunk producer.
-    Requested(Vec<CodeHash>),
+    Requested(BTreeSet<CodeHash>),
```

Comment:
To make it explicit that hashes are unique (based on the discussions during the offsite about the types indicating the certain assumptions, such as uniqueness). I did not change the protocol/network messages but only the internal representation.

Reply:
Makes sense, but in this case I would still keep it as `HashSet` instead of `BTreeSet`, since BTreeSet implies ordering which is not important here, so it can be a bit confusing. Not a strong opinion, feel free to keep it as it is.